functional/SyntheticGCWorkload/playlist.xml

```diff
@@ -146,4 +146,25 @@
 		</groups>
 		<disabled>TODO: create test for all impls, check for and remove any hard-coded J9 specific options</disabled>
 	</test>
+	<test>
+                <testCaseName>SyntheticGCWorkload_DoubleMap_J9</testCaseName>
+                <variations>
+                        <variation>-Xmx2g -Xms2g -Xdump:none -Xgcpolicy:balanced -verbose:gc -XXgc:enableDoubleMapping</variation>
+                </variations>
+                <command>mkdir -p $(REPORTDIR); \
+        cd $(TEST_RESROOT); \
+        $(JAVA_COMMAND) $(JVM_OPTIONS) -cp .:SyntheticGCWorkload.jar net.adoptopenjdk.casa.workload_sessions.Main config/config_doubleMapStress.xml -n; \
```

Comment:
Instead of `.:SyntheticGCWorkload.jar `, please use `$(Q)$(TEST_RESROOT)$(D)SyntheticGCWorkload.jar$(Q)`

Reply:
changed